float64 target_lat			# Info of current and target latitudes and longitudes
float64 target_long
uint8 target_type
uint8 TARGET_TYPE_GPS_ONLY = 0
uint8 TARGET_TYPE_VISION = 1
